source/install/prod-docker.rst

```diff
@@ -46,6 +46,13 @@ Additional Guides:
 
 - **Setup Database Backup** following the `database backup instructions. <https://github.com/mattermost/mattermost-docker/#database-backup>`_
 
+
+Production Docker Setup on Arch Linux
+------------------------------------------------------------
+
+`Use the following online guide from the Arch Linux wiki<https://wiki.archlinux.org/index.php/Mattermost>`_.
```

Comment:
Can you change this to:

``To install on Arch Linux, see the `installation guide <https://wiki.archlinux.org/index.php/Mattermost#With_Docker>`_ on the Arch Linux wiki.``